acquia/acquia_cms
# ACMS-1371: added missing fornt-page background image

**Motivation**
Fixes #ACMS-1371

**Proposed changes**
Added missing background image

**Alternatives considered**
NA

**Testing steps**
- Install site with this branch 
- Install starter module 
- Verify home page should have background image

**Merge requirements**
